### PR TITLE
feat: add a `/__test_loc__` dockerflow endpoint

### LINKED ISSUE #281: Create a dockerflow endpoint to test location responses
During the roll out of 1.3.0, we encountered a bug with location lookups. It would be useful to have a callback that allowed for minimal location information to be returned.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/CONSVC-1256)

### PR DESCRIPTION
**Pull request by jrconlin**
Closes #281

## Description
Adds a `/__test_loc__` that returns minimal location resolution information.

## Testing

https://<server>/__test_loc__
This will use the `X-FORWARDED-FOR` IP address value 

## Issue(s)

Closes #281

### CODE REVIEW

**Review by hackebrot — DISMISSED** [review]
Hi @jrconlin! 👋🏻 

Do you mind adding a unit test to verify that that the new API endpoint is defined and returns `200 OK` with a non-empty response body?

File: src/adm/tiles.rs [hunk 1]
```diff
@@ -123,6 +123,7 @@ pub fn filtered_dma(exclude: &Option<Vec<u16>>, dma: &u16) -> String {
 
 /// Main handler for the User Agent HTTP request
 ///
+/// This calls out to the
```
[the remark this answers is not in the document]
Comment (jrconlin) [reply src/adm/tiles.rs]: yeah, broken comment from a failed merge.
thanks!

**Review by pjenvey — APPROVED** [review]
